swervedrive/icr/kinematicmodel.py

```diff
@@ -2,6 +2,10 @@
 import numpy as np
 
 
+def cartesian_to_lambda(x, y):
+    return np.reshape(1 / np.linalg.norm([x, y, 1]) * np.array([x, y, 1]), (3, 1))
```

Comment:
Uh, wasn't this in the other PR?

Reply:
It was in the unit tests. Should be removed from there and referenced from here.